### DISCUSSION TITLE
Actions queued, degradation of service

### DISCUSSION THREAD

**Discussion by Divagnz** [community/community#49038] [Actions]
### Select Topic Area

Question

### Body

Does any one else is experiencing Degradations issues in actions?

**Comment by nvborisenko** [comment]
Known issue, track it https://www.githubstatus.com/incidents/qfby3shb6r3w

**Reply by Divagnz** [reply]
Yeah basically caught it first ;)

**Comment by acuD1** [comment]
Hi, I'm bumping this. I have a Run in "Queued" since 3 days. How can I resolve this ? 

Cancel from website nor CLI doesn't work.

I submited a Ticket but no response. It's beginning do be critical !

**Comment by Akash1134** [comment]
Hi @acuD1 , Sorry that you're having issues. I'd recommend that you re-run the Actions that were queued. The incident was resolved on Mar 03, 2023 - 21:14 UTC. Please note that you are not being billed for these runs and new runs are not affected by the incident.

For more details, see the GitHub Status report: [Investigating reports of degraded performance](https://www.githubstatus.com/incidents/sr7zptjd0b07). Thank you for your support and patience.

**Reply by acuD1** [reply]
I just got a response from Github Support team. Thank you ;)
